sycl/test/regression/joint_group_conflict.cpp

```diff
@@ -1,6 +1,6 @@
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -fPIC -DCASE1 %s -c -o %t.1.o
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -fPIC -DCASE2 %s -c -o %t.2.o
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -shared %t.1.o %t.2.o -o %t.so
+// RUN: %clangxx -Xclang -opaque-pointers -fsycl -fsycl-targets=%sycl_triple -fPIC -DCASE1 %s -c -o %t.1.o
+// RUN: %clangxx -Xclang -opaque-pointers -fsycl -fsycl-targets=%sycl_triple -fPIC -DCASE2 %s -c -o %t.2.o
+// RUN: %clangxx -Xclang -opaque-pointers -fsycl -fsycl-targets=%sycl_triple -shared %t.1.o %t.2.o -o %t.so
```

Comment:
The problem is that libclc (or something like it) is compiled with opaque pointers. On the other hand, SPIR-V path seems to use non-opaque pointers so I had to use a force switch here to avoid mismatch. Couldn't use non-opaque because that's incompatible with libclc built.